src/main/webapp/resources/celJS/initCelements.js

```diff
@@ -624,16 +623,15 @@
   window.celMessages.isLoaded = false;
 
   try {
-    var topFrame = top || window;
     new Ajax.Request(window.getCelHost(), {
-      method : 'post',
-      parameters : {
-        xpage : 'celements_ajax',
-        ajax_mode : 'Messages'
+      method: 'post',
+      parameters: {
+        xpage: 'celements_ajax',
+        ajax_mode: 'Messages'
       },
-      onSuccess : function(transport) {
+      onSuccess: function (transport) {
         if (transport.responseText.isJSON()) {
-          var newMessages = transport.responseText.evalJSON();
+          let newMessages = transport.responseText.evalJSON();
```

Comment:
`newMessages` is never reassigned
```suggestion
          const newMessages = transport.responseText.evalJSON();
```